e2e/ansible/openebs-on-premise-deployment-guide.md

```diff
@@ -24,10 +24,19 @@ The following instructions have been verified on:
 
 - At least 3 linux machines of above type, i.e., either VMs or bare-metal, if deploying the setup in a hyperconverged mode 
```

Comment:
hyperconverged = hyper converged

Reply:
@mahebbar , this is a standard usage/used as a single node